index.html

```diff
@@ -1365,16 +1433,16 @@ <h2>Dereferencing the Fragment</h2>
 			on the media type ([[RFC2046]]) of the resource, i.e., on the result of
 			<a href="#dereferencing-algorithm-resource"></a>.</p>
 			<ol class="algorithm">
-				<li>If the result of <a href="#dereferencing-algorithm-resource"></a> is an <var>output <a>service endpoint</a> URL</var>,
+				<li>If the result of <a href="#dereferencing-algorithm-resource"></a> is a <var>selected <a>service endpoint</a> URL</var>,
 				and the <var>input <a>DID URL</a></var> contains a <a>DID fragment</a>:
 				<pre class="example nohighlight">did:example:1234?service=files&relativeRef=%2Fmyresume%2Fdoc%3Fversion%3Dlatest#intro</pre>
 				<ol class="algorithm">
-					<li>If the <var>output <a>service endpoint</a> URL</var>
+					<li>If the <var>select <a>service endpoint</a> URL</var>
```

Comment:
Wondering if this should be selected throughout this algorithm
```suggestion
					<li>If the <var>selected <a>service endpoint</a> URL</var>
```